tutorials/full-stack-application/app/app/globals.css

```diff
@@ -0,0 +1,103 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+:root {
+  --foreground-rgb: 0, 0, 0;
+  --background-end-rgb: 255, 255, 255;
+}
+
+@media (prefers-color-scheme: dark) {
+  :root {
+    --foreground-rgb: 255, 255, 255;
+    --background-start-rgb: 0, 0, 0;
+    --background-end-rgb: 0, 0, 0;
+  }
+}
+
+body {
+  color: rgb(var(--foreground-rgb));
+  background: linear-gradient(
+      to bottom,
+      transparent,
+      rgb(var(--background-end-rgb))
+    )
+    rgb(var(--background-start-rgb));
+}
+
+@layer utilities {
+  .text-balance {
+    text-wrap: balance;
+  }
+}
+
+/* Home styling */
+.home {
+    @apply container mx-auto px-2 pt-8
+}
+
+.home_header {
+    @apply sticky top-0 w-full max-w-7xl items-center justify-between font-sans text-sm lg:flex;
+}
+
+.home_header_content {
+    @apply fixed bottom-0 left-0 flex h-48 w-full items-end justify-center bg-gradient-to-t from-white via-white dark:from-black dark:via-black lg:static lg:size-auto lg:bg-none;
+}
+
+.title_container {
+    @apply max-w-2xl mx-auto m-10 pl-4;
+}
+
+/* ----------- */
+
+/* Menu styling */
+.home_menu {
+    @apply max-w-2xl mx-auto mt-10;
+}
+
+.menu-item_card {
+    @apply flex flex-col bg-white rounded-lg shadow-md border m-5;
+}
+
+.menu-item_title {
+    @apply items-center mb-1 p-1 w-full border-b-2 border-gray-100;
+}
+
+.menu-item_title_text {
+    @apply text-[22px] font-bold capitalize;
```

Comment:
I think this needs some color to it, its white-on-white for me so I can't see the coffee name at all

Reply:
Ah... a dark-theme user.